### PR TITLE
Websites dnx core 5.0

### PR DESCRIPTION
**Pull request by jianghaolu**

### CODE REVIEW

File: src/ResourceManagement/WebSite/Microsoft.Azure.Management.Websites/project.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+  "version": "1.0.0-preview",
```
Comment (jianghaolu) [inline src/ResourceManagement/WebSite/Microsoft.Azure.Management.Websites/project.json]: @naveedaz Is it okay we bump up the version here from 0.16.8-preview to 1.0.0-preview?

Comment (naveedaz) [inline src/ResourceManagement/WebSite/Microsoft.Azure.Management.Websites/project.json]: We need to bump the version anyways right? I think it should be fine.

Comment (jianghaolu) [inline src/ResourceManagement/WebSite/Microsoft.Azure.Management.Websites/project.json]: Sounds good. Thanks for the feedback!

**Comment by jianghaolu** [comment]
@azuresdkci retest this please

**Pull request merged**